homesensorhub/communication/data_sender.py

```diff
@@ -48,9 +48,7 @@ def send_data(self, data):
 
     def send_current(self, value, topic):
         topic = self.HOST + "/" + topic + "/current"
-        payload = value #TODO: This give us invalid syntax: f'{value:.3f}'
-        logging.info("Sending data to :" + topic + " --> " + str(payload))
-
+        payload = str(value)  # TODO: This give us invalid syntax: f'{value:.3f}
```

Comment:
This should be converted to a limited number of decimal points to be easier to interpret on the other side.

Reply:
You mean the actual value read by the sensor, right? The payload is of the form:

{'type': 'temperature', 'name': <class 'adafruit_bme280.Adafruit_BME280_I2C'>, 'value': 29.604296875, 'timestamp': datetime.datetime(2020, 8, 1, 15, 26, 9, 29915), 'measurement': 'celsius'}.

Reply:
I will include this in #10.